pkg/nodes/nodes.go

```diff
@@ -97,6 +105,50 @@ func (builder *Builder) ExternalIPv4Networks() ([]string, error) {
 	return ipV4ExternalAddresses, nil
 }
 
+// WaitToBeAllReadyFor waits for all nodes to be Ready for a time duration up to the timeout.
+func (builder *Builder) WaitToBeAllReadyFor(timeout time.Duration) (bool, error) {
```

Comment:
Same comment as https://github.com/openshift-kni/eco-goinfra/pull/101/files#r1317940396, sorry if I wasn't clear

Reply:
I answer on 101 thread, I don't think we need additional wait in this case. this will increase the execution time of this method